### ISSUE TITLE
Edition search to be visible also to librarians (not just super librarians)

### ISSUE THREAD

**Issue by RayBB** [internetarchive/openlibrary#9054]
### Describe the problem that you'd like solved

Right now only super librarians  can see the edition level search button (which is helpful for them when merging works).
We want to change that so it works for librarians too!

### Proposal & Constraints

It should be approximately a 1 line change!

**Comment by seabelis** [comment]
@RayBB Edition search is the default for non-libraians. I think it's works search you want to make visible.

**Comment by seabelis** [comment]
Edition search is the default for non-librarians.

**Comment by quackers2** [comment]
Hello, I am a college student interested in working on more open-source projects; could I be assigned to this issue?

**Comment by RayBB** [comment]
@quackers2 you are assigned. Please let me know if you need any help :)

**Comment by RayBB** [comment]
@quackers2 are you still working on this?

**Comment by quackers2** [comment]
Yes, sorry. I've been a bit busy with work lately but I will get it as soon as I can!

**Comment by quackers2** [comment]
Hey @RayBB, I just created a PR that closes this issue. It looks like I'm not able to assign a reviewer so I just wanted to let you know :D

**Comment by Mones26** [comment]
Hey @RayBB , I worked with @quackers2 on this PR and we were interested in helping with another issue on the project. I personally filled out the Volunteering @ Open Library form but haven't heard back at all. We would like to join the slack so that we can better communicate with the group as part of our school project requirements.

**Comment by RayBB** [comment]
@mekarpeles can you check the volunteer form?

**Comment by mekarpeles** [comment]
Yes!
